tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -89,6 +89,7 @@ def run_pipeline_test(self, speech_recognizer, examples):
             outputs = speech_recognizer(audio)
             self.assertEqual(outputs, {"text": ANY(str)})
         elif "Whisper" in speech_recognizer.model.__class__.__name__:
+            audio.pop("stride")
```

Comment:
This file should not be modified no?

Reply:
In Whisper case we don't want to pass stride as per:

https://github.com/huggingface/transformers/blob/66459ce31967b5dfd955471e5e1ff295c94d5af8/src/transformers/pipelines/automatic_speech_recognition.py#L516-L518

My understanding is that this test path never run before (because of the missing fast tokenizer), so this never came up.

Reply:
No, this test was update in the `main`, you should probably rebase. 😉 Let's revert this change

Reply:
Ah it passes after syncing with main, thanks!